tests/test_end_to_end.py

```diff
@@ -36,6 +37,10 @@
 
 MISSING_THREADS = platform.machine() != 'x86_64'
 
+IS_DOCKER = False
+""" Returns: True if running in a Docker container, else False """
+with open('/proc/1/cgroup', 'rt') as ifh:
```

Comment:
Maybe wrap this in a try/except? This might be fine as-is but, I'm imagining if for some reason the file had permissions 0440.